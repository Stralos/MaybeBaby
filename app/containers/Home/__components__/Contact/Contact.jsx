/* @flow */
import React, { PureComponent } from 'react';
import styled from 'styled-components';

export const Lable = styled.div`
`;
export const ContactInformation = styled.div`
`;

export type Props = {
  lable: string,
  contactInformation: string
}

class Contact extends PureComponent<Props> {
  setContactInformation = () => {
    const { contactInformation } = this.props;
    return {
      __html: contactInformation,
    };
  }

  render = () => {
    const { lable } = this.props;
    return (
      <div>
        <Lable>
          {lable}
        </Lable>
        <ContactInformation dangerouslySetInnerHTML={this.setContactInformation()} />
      </div>
    );
  }
}

export default Contact;
