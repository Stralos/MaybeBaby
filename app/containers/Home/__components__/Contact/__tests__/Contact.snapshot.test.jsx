/* @flow */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Contact from '../Contact';
import type { Props } from '../Contact';

const getProps = (): Props => {
  return {
    label: '',
    contactInformation: '',
  };
};

describe('Contact Component Snapshot', () => {
  it('renders correctly', () => {
    const props = getProps();
    const tree = renderer.create(
      <Contact {...props} />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
