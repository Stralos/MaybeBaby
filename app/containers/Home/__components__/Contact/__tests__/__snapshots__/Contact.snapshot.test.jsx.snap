// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Component Snapshot renders correctly 1`] = `
<div>
  <div
    className=""
  >
    
  </div>
  <div
    className=""
    dangerouslySetInnerHTML={
      Object {
        "__html": "",
      }
    }
  />
</div>
`;
